examples/connext_dds/dynamic_data_request_reply/cpp/PrimesType.cxx

```diff
@@ -0,0 +1,274 @@
+#include <iostream>
+#include <ndds_cpp.h>
+
+
+using namespace std;
+
+/*****************************************************************************/
+/*****************************************************************************/
+/************** This file defines the type for following commented Idl *******/
+/*****************************************************************************/
+/*****************************************************************************/
+
+/* --- Request type: ------------------------------------------------------- */
+
+/* 
+ * This is the request type. 
+ *
+ * It simply contains an integer that requests all the prime numbers below it.
```

Comment:
I agree with you on this. The problem here is that I wanted to make this example compatible with the Request/Reply shipped example (`rti_workspace/6.0.0/examples/connext_dds/c++/hello_world_request_reply`).
If I add this integer then the types will not match...
Do you agree with my approach? (I am doing so because I already had a Support case with this use case)